django_spanner/schema.py

```diff
@@ -123,6 +130,13 @@ def create_model(self, model):
                 self.create_model(field.remote_field.through)
 
     def delete_model(self, model):
+        """
+        Drop the model’s table in the database along with any unique constraints
```

Comment:
Looks like this is a [unicode RIGHT SINGLE QUOTATION MARK](https://www.fileformat.info/info/unicode/char/2019/index.htm), did you write these docstrings in a rich text editor first?

Reply:
Fixed it.